docs/products/opensearch/dashboards/getting-started.rst

```diff
@@ -0,0 +1,119 @@
+Getting started
+===============
+
+To start using Aiven for OpenSearch Dashboards, you should :doc:`create Aiven for OpenSearch service first</docs/products/opensearch/getting-started>` and OpenSearch Dashboards service will be created alongside it. Once the Aiven for OpenSearch service is running you can find connection information to your OpenSearch Dashboards in the service overview page.
```

Comment:
What about "Os Dashboards are part of Aiven for OpenSearch, after creating such service OpenSearch Dashboard information are available in the service overview"

Reply:
Should we also mention that you can open OS dashbaords via your favourite browser?

Reply:
Mmm, I've tried replacing with "Os Dashboards are part of Aiven for OpenSearch, after creating such service OpenSearch Dashboard information are available in the service overview", but it sounds a bit less direct, so I decided to keep the original phrase.

Added the part with the favourite browser!